### PR TITLE
Update secure-your-business-data.md

### PR DESCRIPTION
**Pull request by msbemba**
added steps for MFA

### CODE REVIEW

**Comment by mypil** [comment]
@Ajayan1008 - we can let @JohanFreelancer9 do the copy/edit review for this PR. 

@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by Ajayan1008** [comment]
Thanks, Joyce.

File: Office365-Admin/security-and-compliance/secure-your-business-data.md [hunk 1]
```diff
@@ -55,17 +55,16 @@ To see a training video for how to set up MFA and how users complete the set up,
   
 To set up multi-factor authentication:
 
-1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Setup**.
+1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Users** and **Active Users**.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Users** > **Active Users**.
```

File: Office365-Admin/security-and-compliance/secure-your-business-data.md [hunk 2]
```diff
@@ -55,17 +55,16 @@ To see a training video for how to set up MFA and how users complete the set up,
...
 
-2. In the **Sign-in and security** section, under **Turn on multi-factor authentication (MFA)**, select **View**. 
+2. In the **Active Users** section, Click on  **multi-factor authentication**.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
2. In the **Active Users** section, click on  **multi-factor authentication**.
```

File: Office365-Admin/security-and-compliance/secure-your-business-data.md [hunk 3]
```diff
@@ -55,17 +55,16 @@ To see a training video for how to set up MFA and how users complete the set up,
...
 
-3. On the **Make sign-in more secure** page, select **Get started**.
+3. On the **Multi-factor authentication** page, select **user** if you are enabling this for one user Or you can peform a **Bulk Update**.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
3. On the **Multi-Factor Authentication** page, select **User** if you are enabling this for one user Or you can perform a **Bulk Update**.
```

File: Office365-Admin/security-and-compliance/secure-your-business-data.md [hunk 4]
```diff
@@ -55,17 +55,16 @@ To see a training video for how to set up MFA and how users complete the set up,
...
 
-4. Select the **Require multi-factor authentication for admins** and **Require users to register for multi-factor authentication and block access if risk is detected** check boxes.
+4. CLick on **Enable** under **Quick Steps**
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
4. Select  **Enable** under **Quick Steps** 
```

File: Office365-Admin/security-and-compliance/secure-your-business-data.md [hunk 5]
```diff
@@ -55,17 +55,16 @@ To see a training video for how to set up MFA and how users complete the set up,
...
+2. In the **Active Users** section, Click on  **multi-factor authentication**.
 
-3. On the **Make sign-in more secure** page, select **Get started**.
+3. On the **Multi-factor authentication** page, select **user** if you are enabling this for one user Or you can peform a **Bulk Update**.
 
-4. Select the **Require multi-factor authentication for admins** and **Require users to register for multi-factor authentication and block access if risk is detected** check boxes.
+4. CLick on **Enable** under **Quick Steps** 
 
-5. Under **Do you want to exclude anyone from these policies**, select any users that you want to exclude from the drop-down list box.
+5. In the Pop-up window, CLickc on **Enable Multi-Factor Authentication**
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
5. In the pop-up window, choose **Enable Multi-Factor Authentication**.
```

**Comment by JohanFreelancer9** [comment]
@msbemba It looks like you did not commit my last two edits. Could you please check?

**Comment by msbemba** [comment]
> @msbemba It looks like you did not commit my last two edits. Could you please check?

I'm sure I added the suggestion. Can you add the commit again? I'll do that right away.,

File: Office365-Admin/security-and-compliance/secure-your-business-data.md [hunk 6]
```diff
@@ -55,17 +55,16 @@ To see a training video for how to set up MFA and how users complete the set up,
...
+1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Users** > **Active Users**.
 
-2. In the **Sign-in and security** section, under **Turn on multi-factor authentication (MFA)**, select **View**. 
+2. In the **Active Users** section, select  **Multi-Factor Authentication**.
 
-3. On the **Make sign-in more secure** page, select **Get started**.
+3. On the **Multi-Factor Authentication** page, select **User** if you are enabling this for one user Or you can perform a **Bulk Update**.
 
-4. Select the **Require multi-factor authentication for admins** and **Require users to register for multi-factor authentication and block access if risk is detected** check boxes.
+4. Click on **Enable** under **Quick Steps**.
```
Comment (mypil) [inline Office365-Admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
4. Select **Enable** under **Quick Steps**.
```

File: Office365-Admin/security-and-compliance/secure-your-business-data.md [hunk 7]
```diff
@@ -55,17 +55,16 @@ To see a training video for how to set up MFA and how users complete the set up,
...
 
-5. Under **Do you want to exclude anyone from these policies**, select any users that you want to exclude from the drop-down list box.
+5. In the Pop-up window, CLickc on **Enable Multi-Factor Authentication**.
```
Comment (mypil) [inline Office365-Admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
5. In the pop-up window, choose **Enable Multi-Factor Authentication**.
```

**Comment by msbemba** [comment]
> @msbemba - I added the last 2 copy/edit changes as suggested by @JohanFreelancer9. Please merge these commits. Thank you.

Done.

**Comment by mypil** [comment]
@Sirkkuw @Ajayan1008 - PR has been copy edited and is ready for final review. This is based on issue #961. Please take note that the metadata author has been updated as discussed via email.

Thank you.

cc: @kenwith

**Pull request merged**